src/default-cache-middleware.js

```diff
@@ -15,15 +15,21 @@ const _ = require('lodash');
 module.exports = function defaultCacheMiddleware(ctx, next) {
   let cacheCheck, cacheHit = false;
   if (ctx.req.method.toLowerCase() === 'get' && !ctx.isCollection) {
-    // TODO: use ctx.req.url as cache key and add json() method to cached result: https://oktainc.atlassian.net/browse/OKTA-280090
-    cacheCheck = ctx.cacheStore.get(ctx.req.uri)
+    cacheCheck = ctx.cacheStore.get(ctx.req.url)
```

Comment:
note: `ctx.req.uri` was a typo and so the value was always undefined. `ctx.req.url` holds a string value for the original uri of the request.